gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/CommonSteps.cs

```diff
@@ -244,6 +245,16 @@ private static object ToT(string enumName, string graphName)
             return T.GetByValue(enumName);
         }
 
+        private static object ToDirection(string enumName, string graphName)
+        {
+            return Direction.GetByValue(enumName);
+        }
+
+//        private static string toNormalCase(string name)
```

Comment:
(nitpick) Why this commented out code?

Reply:
it was just some left over code from trying to debug a problem - removed it and force pushed.